war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+    *       name of the new organization to create
+    */
+   public void createOrganization(final String organizationId, final String organizationName) {
+      DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, organizationId)
+            .append(LumeerConst.Organization.ATTR_ORG_NAME, organizationName)
+            .append(LumeerConst.Organization.ATTR_ORG_DATA, new DataDocument())
+            .append(LumeerConst.Organization.ATTR_USERS, new DataDocument());
+      dataStorage.createDocument(LumeerConst.Organization.COLLECTION_NAME, document);
+   }
+
+   /**
+    * Renames a name of the given organization according to its id.
+    *
+    * @param organizationId
+    *       id of the given organization
+    * @param newOrganizationName
+    *       new name of the organization
+    */
+   public void renameOrganization(final String organizationId, final String newOrganizationName) {
+      DataDocument dataDocument = new DataDocument(LumeerConst.Organization.ATTR_ORG_NAME, newOrganizationName);
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, dataDocument, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Drops the organization according to its id.
+    *
+    * @param organizationId
+    *       id of the given organization to drop
+    */
+   public void dropOrganization(final String organizationId) {
+      dataStorage.dropDocument(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Reads additional information about the given organization.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @return DataDocument object including additional organization info
+    */
+   public DataDocument readOrganizationInfoData(final String organizationId) {
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_ORG_DATA));
+      return document != null ? document.getDataDocument(LumeerConst.Organization.ATTR_ORG_DATA) : null;
+   }
+
+   /**
+    * Reads specified additional information about the given organization.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param infoDataAttribute
+    *       attribute of the organization additional info
+    * @return value of the organization info attribute
+    */
+   public String readOrganizationInfoData(final String organizationId, final String infoDataAttribute) {
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_ORG_DATA));
+      DataDocument infoDataDocument = document.getDataDocument(LumeerConst.Organization.ATTR_ORG_DATA);
+      return infoDataDocument != null ? infoDataDocument.getString(infoDataAttribute) : null;
```

Comment:
You should use previous function `readOrganizationInfoData` here, because it has duplicated implementation.